app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -119,6 +124,23 @@ public function store(Request $request, $id)
             $return_msg = trans('admin/users/message.declined');
         }
 
+        $item = $acceptance->checkoutable_type::find($acceptance->checkoutable_id);
+
+        $data = [
+            'item' => $item,
+            'eula' => $item->getEula(),
+            'signature' => storage_path().'/private_uploads/signatures/'.$sig_filename,
+            'logo' => public_path().'/uploads/snipe-logo.png',
+        ];
+
+        \Log::error(storage_path().'/eula-pdfs/'.$sig_filename);
+
+        $pdf = Pdf::loadView('account.accept.accept-eula', $data);
+        $stored_eula = Storage::put('private_uploads/eula-pdfs/accepted-eula-'.date('Y-m-d-h-i-s').'.pdf', $pdf->output());
+
+        //not sure what im doing here,but I think its something of this.
+        route('log.storedeula.download', $stored_eula);
```

Comment:
I want to send the value safely to another file so this is where my brain is starting with this.